src/btree/bt_huffman.c

```diff
@@ -336,6 +336,7 @@ __wt_huffman_read(WT_SESSION_IMPL *session, WT_CONFIG_ITEM *ip,
 		WT_ERR(__wt_getline(session, fs, tmp));
 		if (tmp->size == 0)
 			break;
+		/* NOLINTNEXTLINE(cert-err34-c) */
```

Comment:
Clang Tidy doesn't like the use of sscanf to convert strings to integers as it supposedly won't report any conversion errors (it suggests to use strtoul instead).

Some of the sscanf usages seemed a bit complex so I wasn't confident replacing them without understanding the codebase better. So I've decided to just suppress the linter warnings for the time being.